src/components/PostInfo/PostInfo.jsx

```diff
@@ -1,3 +1,40 @@
-export const PostInfo = () => (
-  <>Put the post here</>
+import { UserInfo } from '../UserInfo';
+
+export const PostInfo = ({ post }) => (
+  <div className="PostInfo" key={post.id}>
+    <div className="PostInfo__header">
+      <h3 className="PostInfo__title">{post.title}</h3>
+
+      <p>
+        {' Posted by  '}
+
+        <UserInfo user={post.user} />
```

Comment:
Show `UserInfo` only if user is exist

```suggestion
        {post.user && <UserInfo user={post.user} />}
```

Reply:
Not fixed